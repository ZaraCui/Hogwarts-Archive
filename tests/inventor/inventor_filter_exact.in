ADD COLLECTION tests/data/books_small.csv
INVENTOR Minerva McGonagall
EXIT

