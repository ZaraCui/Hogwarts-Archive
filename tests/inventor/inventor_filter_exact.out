user: 12 spellbooks successfully added.

user: Transfiguration I (Minerva McGonagall)
Transfiguration i (Minerva McGonagall)

user: Ending Archive process.
